src/core/public/core_system.ts

```diff
@@ -27,6 +27,7 @@ import { InjectedMetadataParams, InjectedMetadataService } from './injected_meta
 import { LegacyPlatformParams, LegacyPlatformService } from './legacy_platform';
 import { LoadingCountService } from './loading_count';
 import { NotificationsService } from './notifications';
+import { UICapabilitiesService } from './ui_capabilities';
```

Comment:
It looks like we have a mismatch of `UI` here to `Ui` below, it is safe to assume that we want everything to be `Ui` in the new platform?

Reply:
I don't think we have any formal agreement on that, I personally prefer abbreviations to be all caps like is done here.